server/schema.ts

```diff
@@ -137,6 +137,9 @@ export interface IUser {
 
 	admin?: boolean;
 
+	uuid?: string;
```

Comment:
We should update existing users to include a uuid when we merge this so please make this non-optional